recipes/clingo/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = 'clingo' %}
+{% set version = "5.4.0" %}
+{% set build = 0 %}
```

Comment:
If variables are only used once it's generally clearer to just write the value directly
```suggestion
```